### PR TITLE
refactor audio to container

### PR DESCRIPTION
**Pull request by BaconPancakes**

### CODE REVIEW

File: client/src/components/GamePortalContainer/GamePortalContainer.jsx [hunk 1]
```diff
@@ -2,15 +2,295 @@ import React, { Component } from 'react';
 import Tone from 'tone';
 import Grid from '@material-ui/core/Grid';
 import Container from '@material-ui/core/Container';
-import { ChatMessageBox, GameInfoTable, Microphone, AudioDisplayTable, SampleLooper, Looper, UserRecorder } from '..';
+import Button from '@material-ui/core/Button';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import { ChatMessageBox, GameInfoTable, Microphone, AudioDisplayTable } from '..';
+// import Game from '../../utils/GameModel';
```
Comment (shannonphu) [inline client/src/components/GamePortalContainer/GamePortalContainer.jsx]: removed commented out

File: client/src/components/GamePortalContainer/GamePortalContainer.jsx [hunk 2]
```diff
@@ -2,15 +2,295 @@ import React, { Component } from 'react';
...
+        this.createClickTrack();
+        this.createMetronome();
+
+    }
+    // -------------------------------------------------------------------------------------
+    // Tone Object Creation
+
+    createMetronome() {
+        let metronomeSynth = new Tone.MembraneSynth().toMaster();
+        metronomeSynth.volume.value = 5;
```
Comment (shannonphu) [inline client/src/components/GamePortalContainer/GamePortalContainer.jsx]: maybe this should be controlled by the volume controller, so let the master volume set it?

File: client/src/components/GamePortalContainer/GamePortalContainer.jsx [hunk 3]
```diff
@@ -2,15 +2,295 @@ import React, { Component } from 'react';
...
+        this.clickTrack = new Tone.Loop(
+            (time) => {woodBlock.start(time, 0, '4n');},
+            '4n'
+        );
+    }
+
+    createLoopPlayer() {
+        // check for loop audio
+        if (this.props.game.baselinePlayer != null) {
+            var baselinePlayerName = this.props.game.baselinePlayer.playerName;
```
Comment (shannonphu) [inline client/src/components/GamePortalContainer/GamePortalContainer.jsx:97]: you declare this here in this if statement scope? do you use it elsewhere

File: client/src/components/Looper/UserRecorder.jsx [hunk 4]
```diff
@@ -158,30 +153,59 @@ class UserRecorder extends Component {
...
+                availableAudio[playerName] = playerData.audioUrl;
+            }
+        });
+        let loopUrl = null;
+        Object.keys(this.props.room.users).map((playerName, i) => {
+            let playerData = this.props.room.users[playerName];
+            if (playerData.loopUrl != null) {
+                loopUrl = playerData.loopUrl;
+            }
+        });
```
Comment (shannonphu) [inline client/src/components/Looper/UserRecorder.jsx]: Cant you set loopUrl in the above loop too?

**Review by shannonphu — CHANGES_REQUESTED** [review]

**Comment by BaconPancakes** [comment]
> you declare this here in this if statement scope? do you use it elsewhere 

javascript is weird yoo

**Pull request merged**